scripts/pr-merge-autopilot.sh

```
@@ -160,42 +158,238 @@ fi
+    for bot_login in "${BOT_LOGINS[@]}"; do
+      local bot_base="${bot_login%\[bot\]}"
+      [[ "$author_base" == "$bot_base" ]] && is_bot=true && break
+    done
+
+    [[ "$is_bot" != "true" ]] && continue
+
+    local thread_id
+    thread_id=$(echo "$thread_json" | jq -r '.id')
+
+    if [[ "$dry_run" == "false" ]]; then
+      gh api graphql -f query='mutation($threadId:ID!, $body:String!) { addPullRequestReviewComment(input:{pullRequestReviewThreadId:$threadId, body:$body}) { comment { id } } }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)